k0765065/assignment07.py

```diff
@@ -0,0 +1,10 @@
+from sys import argv
+
+if len(argv) == 3:
+    from prancercise import prancercise
+    script, string, repeat = argv
+    output = prancercise(string, repeat)
+    print output
+    
+else:
+    print("Please input a statement and a surrounding character.")
```

Comment:
Here you should get the input from the user ( `raw_input` ) and then use it to call prancercise